array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,12 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length - count; i++) {
+    if (array[i] === newArray[i]) {
+      return newArray;
```

Comment:
You should be able to delete these lines, because the main condition is `i < array.length`.